tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/RuntimeBrokerTests.cs

```diff
@@ -121,9 +121,9 @@ public async Task WamUsernamePasswordRequestAsync()
             MsalAssert.AssertAuthResult(result, TokenSource.Cache, labResponse.Lab.TenantId, expectedScopes);
 
             // Acquire token interactively WithAccount
-            result = await pca.AcquireTokenInteractive(scopes).WithAccount(account).ExecuteAsync().ConfigureAwait(false);
+            //result = await pca.AcquireTokenInteractive(scopes).WithAccount(account).ExecuteAsync().ConfigureAwait(false);
```

Comment:
forgotten comment?

Reply:
No, this test is flaky. It pops up a WAM UI when run locally. On DevOps build, I noticed that when both net6 and netcore3.1 targets were specified for the test project, this test would hang here when it's run the second time...
I added a comment why I commented it out.